test/init_test.go

```diff
@@ -291,15 +291,6 @@ func getCRDYaml(ctx context.Context, cs *clients, ns string) ([]byte, error) {
 		printOrAdd(i)
 	}
 
-	v1alpha1Runs, err := cs.V1alpha1RunClient.List(ctx, metav1.ListOptions{})
```

Comment:
I may be missing some context here.

If this is panicking, shouldn't we try to fix the panic instead of removing this part of the test altogether? (my assumption is because the client isn't present?)

Reply:
Thanks Billy. I was mistaken thought that with the removal of clients setup at https://github.com/tektoncd/pipeline/blob/a7424c4885dbaab56a7a31cb9f98508a6f7fc3fd/test/clients.go#L101-L112 for TEP114, this would not be necessary moving forward.

Reply:
when running the following e2e test on local branch the following error occured.
Currently debugging through the reason why it failed but I think the removed line 294 in init_test could be the reason it failed

```
± |v1-storage → origin ↑1 ↓2 {51} U:2 ✗| → go test -v -tags=e2e -count=1 ./test -run ^TestLargerResultsSidecarLogs
=== RUN   TestLargerResultsSidecarLogs
=== PAUSE TestLargerResultsSidecarLogs
=== CONT  TestLargerResultsSidecarLogs
=== RUN   TestLargerResultsSidecarLogs/larger_results_via_sidecar_logs
=== PAUSE TestLargerResultsSidecarLogs/larger_results_via_sidecar_logs
=== CONT  TestLargerResultsSidecarLogs/larger_results_via_sidecar_logs
    larger_results_sidecar_logs_test.go:69: Create namespace arendelle-qtjd7 to deploy to
    larger_results_sidecar_logs_test.go:69: Verify SA "default" is created in namespace "arendelle-qtjd7"
    larger_results_sidecar_logs_test.go:74: Setting up test resources for "larger results via sidecar logs" test in namespace arendelle-qtjd7
    larger_results_sidecar_logs_test.go:82: Waiting for PipelineRun larger-results-sidecar-logs in namespace arendelle-qtjd7 to complete
    larger_results_sidecar_logs_test.go:84: Error waiting for PipelineRun larger-results-sidecar-logs to finish: "larger-results-sidecar-logs" failed
    panic.go:522: ############################################
    panic.go:522: ### Dumping objects from arendelle-qtjd7 ###
    panic.go:522: ############################################
--- FAIL: TestLargerResultsSidecarLogs (14.11s)
    --- FAIL: TestLargerResultsSidecarLogs/larger_results_via_sidecar_logs (14.11s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x38 pc=0x103dfed20]
 
goroutine 12 [running]:
testing.tRunner.func1.2({0x1044ec4a0, 0x105cf9360})
	/usr/local/go/src/testing/testing.go:1526 +0x1c8
testing.tRunner.func1()
	/usr/local/go/src/testing/testing.go:1529 +0x384
panic({0x1044ec4a0, 0x105cf9360})
	/usr/local/go/src/runtime/panic.go:884 +0x204
github.com/tektoncd/pipeline/test.getCRDYaml({0x104924b98, 0x1400036fd10}, 0x14000328b60, {0x14000304d30, 0xf})
	/Users/jeromeju/go/src/github.com/tektoncd/pipeline/test/init_test.go:294 +0xab0
github.com/tektoncd/pipeline/test.tearDown({0x104924b98, 0x1400036fd10}, 0x14000573520, 0x14000328b60, {0x14000304d30, 0xf})
	/Users/jeromeju/go/src/github.com/tektoncd/pipeline/test/init_test.go:98 +0xc0
runtime.Goexit()
	/usr/local/go/src/runtime/panic.go:522 +0x180
testing.(*common).FailNow(0x14000573520)
	/usr/local/go/src/testing/testing.go:980 +0x44
testing.(*common).Fatalf(0x14000573520, {0x103e9a76d?, 0x14000328b60?}, {0x14000d1fef0?, 0x1b?, 0x0?})
	/usr/local/go/src/testing/testing.go:1064 +0x60
github.com/tektoncd/pipeline/test.TestLargerResultsSidecarLogs.func1(0x14000573520)
	/Users/jeromeju/go/src/github.com/tektoncd/pipeline/test/larger_results_sidecar_logs_test.go:84 +0x4b8
testing.tRunner(0x14000573520, 0x14000945290)
	/usr/local/go/src/testing/testing.go:1576 +0x10c
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1629 +0x368
FAIL	github.com/tektoncd/pipeline/test	15.322s
FAIL
```

Reply:
I'm noticing this on other PRs too, for example https://storage.googleapis.com/tekton-prow/pr-logs/pull/tektoncd_pipeline/6516/pull-tekton-pipeline-alpha-integration-tests/1649485487607386112/build-log.txt

It seems like this was introduced by https://github.com/tektoncd/pipeline/pull/6508, but I'm not sure why it is nondeterministic